cities.csv

```diff
@@ -0,0 +1,247 @@
+Lusaka  Zambia,Find one urban park for this city
+Luxembourg  Luxembourg,Find one urban park for this city
+Madrid  Spain,Find one urban park for this city
+Majuro  Marshall Islands,Find one urban park for this city
+Malabo  Equatorial Guinea,Find one urban park for this city
+Malé  Maldives,Find one urban park for this city
+Managua  Nicaragua,Find one urban park for this city
+Manama  Bahrain,Find one urban park for this city
+Manila  Philippines,Find one urban park for this city
+Maputo  Mozambique,Find one urban park for this city
+Marigot  Saint Martin,Find one urban park for this city
+Maseru  Lesotho,Find one urban park for this city
+Mata-Utu  Wallis and Futuna,Find one urban park for this city
+Lobamba  Swaziland,Find one urban park for this city
+Melekeok  Palau,Find one urban park for this city
+Mexico City [e]  Mexico,Find one urban park for this city
+Minsk  Belarus,Find one urban park for this city
+Mogadishu  Somalia,Find one urban park for this city
+Monaco  Monaco,Find one urban park for this city
+Monrovia  Liberia,Find one urban park for this city
+Montevideo  Uruguay,Find one urban park for this city
+Moroni  Comoros,Find one urban park for this city
+Moscow  Russia,Find one urban park for this city
+Muscat  Oman,Find one urban park for this city
+Nairobi Kenya,Find one urban park for this city
+Nassau  Bahamas,Find one urban park for this city
+Naypyidaw  Myanmar,Find one urban park for this city
+N'Djamena  Chad,Find one urban park for this city
+New Delhi  India,Find one urban park for this city
+Niamey  Niger,Find one urban park for this city
+Nicosia  Cyprus,Find one urban park for this city
+Nicosia  Northern Cyprus,Find one urban park for this city
+Nouakchott  Mauritania,Find one urban park for this city
+Nouméa  New Caledonia,Find one urban park for this city
+Nukuʻalofa  Tonga,Find one urban park for this city
+Nuuk  Greenland,Find one urban park for this city
+Oranjestad  Aruba,Find one urban park for this city
+Oslo  Norway,Find one urban park for this city
+Ottawa  Canada,Find one urban park for this city
+Ouagadougou Burkina Faso,Find one urban park for this city
+Pago Pago  American Samoa,Find one urban park for this city
+Palikir  Federated States of Micronesia,Find one urban park for this city
+Panama City  Panama,Find one urban park for this city
+Papeete  French Polynesia,Find one urban park for this city
+Paramaribo  Suriname,Find one urban park for this city
+Paris  France,Find one urban park for this city
+Philipsburg  Sint Maarten,Find one urban park for this city
+Phnom Penh  Cambodia,Find one urban park for this city
+Plymouth Montserrat,Find one urban park for this city
+Podgorica  Montenegro,Find one urban park for this city
+Port Louis  Mauritius,Find one urban park for this city
+Port Moresby  Papua New Guinea,Find one urban park for this city
+Port Vila  Vanuatu,Find one urban park for this city
+Port-au-Prince  Haiti,Find one urban park for this city
```

Comment:
yes, the question is always the same... I do not like it either, but the alternative is to fix it in the template but then users could not add their own questions.
about the more than one space, the original cities file was like that... but I can leave only one space, I think this was not on purpose (some cities have 2 spaces and some others only one...)